style.css

```diff
@@ -0,0 +1,71 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    min-height: 100vh;
+    background: #d3d3d3;
+    font-family: 'Roboto' , sans-serif;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+::-webkit-scrollbar{
+    display: none;
+}
+.paper{
+    position: relative;
+    background: rgba(190, 204, 208, 0.944);
+    width: 90%;
+    max-width: 800px;
+    min-width: 400px;
+    height: 480px;
+    margin: 0 auto;
+    border-radius: 5px;
+    box-shadow: 5px 5px 20px 5px black;
+    overflow: hidden;
+}
+
+.paper::before{
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    width: 60px;
+    background: radial-gradient(grey 6px, transparent 7px)repeat-y;
+    background-size: 30px 30px;
+    border-right: 3px solid green;
+    box-sizing: border-box;
+}
+
+.paper-content{
+    position: absolute;
+    top: 30px;
+    right: 0;
+    bottom: 30px;
+    left: 60px;
+    background: linear-gradient(transparent 29px, green 29px);
+    background-size: 30px 30px;
+}
+
+.paper-content textarea{
+    width: 100%;
+    max-width: 100%;
+    height: 100%;
+    max-height: 100%;
+    line-height: 30px;
+    background: transparent;
+    padding: 0 10px;
+    border: 0;
+    outline: 0;
+    color: blue;
+    font-family: cursive;
+    font-weight: bold;
+    font-size: 20px;
+    box-sizing: border-box;
+    resize: none;
+    z-index: 1;
```

Comment:
is z-index necessary ?

Reply:
in current situation their is no need of it but i put it as for future basis because if we add some more content in this project its automatically will show its value whenever we will give position to it..